app/fixtures/challenges.json

```diff
@@ -0,0 +1,86 @@
+[
+{
+  "model": "app.challenge",
+  "pk": 2,
+  "fields": {
+    "name": "Polar Bear Habitat - Engage users",
+    "description": "The Canadian Polar Bear Habitat is known around the world for its polar bear research and\r\neducation. However, its remote location limits the number of students, teachers, and life-long\r\nlearners that it can reach. In order to connect with more people � and boost its impact the\r\nHabitat would like to be able to deliver a virtual experience. This could include things like\r\nhandouts, interactive activities and graphics, videos, virtual tours, and quizzes as long as\r\nthe user is engaged and can navigate the experience with ease. Here is a dropbox folder with\r\nadditional information and images that you can consult to better understand our challenge\r\nand build your project! Our website is always there for you to visit as well! Good Luck :))",
+    "created": "2020-09-13",
+    "start": "2020-09-13T15:26:34Z",
+    "end": "2020-09-13T15:26:36Z",
+    "image": "",
+    "tags": [
+      3,
+      4,
+      5,
+      6
+    ],
+    "creators": [
+      2
+    ],
+    "rewards": []
+  }
+},
+{
+  "model": "app.challenge",
+  "pk": 3,
+  "fields": {
+    "name": "TechAide",
+    "description": "Centraide is one of the largest investors in the community sector. Each year, Centraide\r\norganizes a fundraising campaign to help with our mission to end poverty and social isolation.\r\nAlthough our fundraising results have increased in the last years, we are struggling to attract\r\nyoung donors. As our more generous donors head into retirement, it will be important to\r\nrecruit young donors to insure income stability. Your goal is to come up with an innovative\r\nsolution that can help us attract more donors, especially the younger generation. You can\r\ncheck out our website to learn more about what we do. Here is some data that you can use to\r\nbuild your project. Thanks!",
+    "created": "2020-09-13",
+    "start": "2020-09-17T15:28:46Z",
+    "end": "2020-09-29T15:28:48Z",
+    "image": "",
+    "tags": [
+      3,
+      5,
+      6,
+      9
+    ],
+    "creators": [
+      1,
```

Comment:
These profile objects need to be in a fixture, as well as their corresponding `User` objects.